firmware/boron-ins-fsm/test/mocks/eeprom.h

```diff
@@ -1,5 +1,17 @@
+/*
```

Comment:
I'm interested in your thoughts on this as well.

Reply:
Hmm, I read the page you linked as well as a few other results on google. What do you think are the main downsides of using a fake for this? The pages I looked at mentioned that both mocks and fakes are used for unit tests, and several also used fake memory as an example of a case where you might use a fake. 

I agree that making a mock that sets return values for EEPROM functions would look similar. Do you think having the fake for other existing and future uses would be less reliable than a mock? i.e. it will catch less errors, etc.